MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -75,6 +75,7 @@ import Foundation
     ///   - forceRefresh: Ignore any existing access token in the cache and force MSAL to get a new access token from the service.
     ///   - correlationId: Optional. UUID to correlate this request with the server for debugging.
     ///   - delegate: Delegate that receives callbacks for the Get Access Token flow.
+    @available(*, deprecated, message: "Use the 'getAccessToken' method that also receives a list of scopes to be included in the token")
     @objc public func getAccessToken(forceRefresh: Bool = false, correlationId: UUID? = nil, delegate: CredentialsDelegate) {
         Task {
```

Comment:
This method and the one with scopes, can (and need IMO) use the same logic.
We should be able to get rid of the `getAccessTokenInternal` method

Reply:
Logic is now in one place.